### ISSUE TITLE
Display of data on chrome for android when protection disabled 

### ISSUE THREAD

**Issue by EdinburghWG** [AdguardTeam/AdGuardHome#6860]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Custom (please mention in the description)

### Installation

GitHub releases or script from README

### Setup

On one machine

### AdGuard Home version

107.46

### Action

Disabled protection on Chrome on Android (Motorola Edge 40). After disabling it, graphs with statistics multiply 😱 and are all over the screen. Enabling protection puts everything back in order.

### Expected result

Expected to see graphs with statistics, each for the relevant area

### Actual result

Stats from one area 'multiply' on the screen

### Additional information and/or screenshots

RPi 4, latest RaspOS

**Comment by EdinburghWG** [comment]
Yes, sure. A photo as I don't know how to do a screenshot on the edge. My old Moto G7 doesn't have that problem.
![17117051741654377213518602120834](https://github.com/AdguardTeam/AdGuardHome/assets/134513751/35d99d19-0a1c-499d-b550-6002b4a87c0d)

**Comment by EdinburghWG** [comment]
Hi

Here it is with the protection on.

![17117109228204313553141827322932](https://github.com/AdguardTeam/AdGuardHome/assets/134513751/797cfb3f-014f-4d76-a247-3faa9053b7f1)

**Comment by EdinburghWG** [comment]
Of course. In the first screenshots, the same graph, based on DNS queries is repeated three times. It's the same blue graph. There's a tiny bit of the second (red) graph in the middle of that picture. On top of everything, the number of block queries (number in red) is repeated twice in that picture. Also, instead of the red graph, a blue one is shown.

In the second screenshot, the first graph is DNS queries, the second one blocked queries. The numbers correspond to both, respectively.

**Comment by Sil3ntVip3r** [comment]
I retested this against the current dashboard implementation.

The affected v0.107.46 dashboard used Nivo.  AdGuard Home replaced Nivo with
Recharts in v0.107.75, and current `master` now uses Chart.js with explicit
chart destruction during cleanup
([source](https://github.com/AdguardTeam/AdGuardHome/blob/5f6cb57df9d54d8a89afc9200d813411f9ce4f7a/client_v2/src/helpers/useChart.ts#L55-L67)).

I also ran a disposable lifecycle regression at a 390-pixel mobile viewport:

- the initial dashboard created exactly four canvases and four chart instances;
- a reactive statistics update reused those same four instances;
- disabling protection did not create any additional chart;
- route-style unmount destroyed all four instances;
- remount created exactly four new instances, and final unmount destroyed those
  as well.

Both lifecycle tests passed.  This proves the duplicate-chart DOM/lifecycle
path is no longer present in the current code, although it cannot reproduce the
reporter's exact Motorola/Chrome GPU environment.

Could someone affected retest with v0.107.75 or newer, preferably the current
release, using Chrome on the original device?  If it still occurs, the Chrome
version and a short screen recording would give us the missing device-specific
evidence.  Otherwise this appears safe to close as obsolete after the chart
implementation replacements.
